### PR TITLE
Feat/add icon svgs

### PR DESCRIPTION
**Pull request by meghaniankov**
# Description

Add icon svgs to static folder
Update drone to copy icons to s3

# Checklist

Pull request contains:

- [ ] A new component
- [ ] Component maintenance: improvement / bug fix / etc
- [ ] Component library change: storybook / webpack / etc

Definition of done:

- [ ] Includes theme changes for both Uswitch and money
- [ ] Work has been tested in multiple browsers
- [ ] PR description includes description of change
- [ ] PR description includes screenshot of change
- [ ] If new component, designer has approved screenshot
- [ ] If the change will affect other teams, that team knows about this change
- [ ] If introducing a new component behaviour, added a story to cover that case.

### CODE REVIEW

**Review by jonathan-fielding — DISMISSED** [review]

File: .drone.yml [hunk 1]
```diff
@@ -60,6 +60,7 @@ pipeline:
     commands:
       - npm config set '//registry.npmjs.org/:_authToken' $NPM_TOKEN
       - npx lerna publish from-package --yes
+      - "aws s3 cp --region eu-west-1 --recursive ./static/icons s3://trustyle-icons --cache-control max-age=604800 --metadata '{\"x-amz-meta-surrogate-control\": \"max-age=604800\"}' --acl=public-read"
```
Comment (jonathan-fielding) [inline .drone.yml:63]: cant be in publish becasue otherwise it tries to do it on pr push

File: .drone.yml [hunk 2]
```diff
@@ -88,3 +88,12 @@ pipeline:
     secrets: [slack_webhook]
     commands:
       - node ./scripts/slack-unpublished.js
+  
+  publish-icons:
+    when:
+      event: push
+      branch: master
+    image: registry.usw.co/cloud/node-12-puppeteer:03228dd2d438979e8973ec16351c4d7bc88f552e
+    secrets: ['npm_token']
```
Comment (jonathan-fielding) [inline .drone.yml:97]: dont need this

Comment (meghaniankov) [reply .drone.yml]: removed

File: .drone.yml [hunk 3]
```diff
@@ -88,3 +88,12 @@ pipeline:
     secrets: [slack_webhook]
     commands:
       - node ./scripts/slack-unpublished.js
+  
+  publish-icons:
+    when:
+      event: push
+      branch: master
+    image: registry.usw.co/cloud/node-12-puppeteer:03228dd2d438979e8973ec16351c4d7bc88f552e
```
Comment (jonathan-fielding) [inline .drone.yml:96]: check this image vs the one in money-includes

Comment (meghaniankov) [reply .drone.yml]: Updated

**Review by jonathan-fielding — APPROVED** [review]

**Pull request merged**